sonar-project.properties

```
@@ -0,0 +1,8 @@
+sonar.projectName=RPE :: expressjs template
+sonar.projectKey=RPEET
```

seems like a weird key?

---

I follow the same naming convention that in I&A.

---

I think this would be better?
```suggestion
sonar.projectKey=rpe-expressjs-template
```